.circleci/config.yml

```diff
@@ -49,7 +49,7 @@ jobs:
           name: 🏁 Build Component Packages & Update Dependencies/Artifacts
           command: |
             python -m venv venv && . venv/bin/activate
-            pip install --upgrade pip wheel
+            pip install --upgrade pip wheel setuptools
```

Comment:
ok the reason this works took me a few days to workout, but basically celery prepends the current working directory to sys.path if it's not already present. This means the local source tree takes priority over installed modules so "import tests" works but "import dash" fails with a missing attribute on dash.dcc

When setuptools and wheel is installed pip will use "setup.py develop" which adds the current sys.path to site-packages/easy-install.pth which *appends* the current directory so the local source tree is available but installed modules take priority, this means "import tests" works and so does "import dash"

Reply:
adding setuptools here is just a workaround that causes pip to fallback to legacy editable project mode, I have a fix for modern editables here https://github.com/plotly/dash/pull/2749